components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/Constants.java

```diff
@@ -241,6 +241,10 @@ public static class ErrorToi18nMappingConstants {
         public static final String INCORRECT_ERROR_MAPPING_KEY = "incorrect.error.mapping";
         public static final String SUSPICIOUS_AUTHENTICATION_ATTEMPTS_SUSPICIOUS_AUTHENTICATION_ATTEMPTS_DESCRIPTION = "suspicious.authentication.attempts_suspicious.authentication.attempts.description";
         public static final String SUSPICIOUS_AUTHENTICATION_ATTEMPTS_SUSPICIOUS_AUTHENTICATION_ATTEMPTS_DESCRIPTION_I18N_KEY = "suspicious.authentication.attempts";
+        public static final String AUTHENTICATION_FAILED_NO_REGISTERED_DEVICE_FOUND =
+                "authentication.failed_no.registered.device.found";
```

Comment:
Is this key correct. According to the https://github.com/wso2/identity-apps/pull/3156, `AUTHENTICATION_FAILED_NO_REGISTERED_DEVICE_FOUND = "authentication.failed"`?

Reply:
In getErrorCodeToi18nMapping method in AuthenticationEndPointUtil class, status and status message is concatenated by an underscore. That's why we need to add the key like this